nservicebus/upgrades/7to8/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Upgrade Version 7 to 8
+summary: Instructions on how to upgrade NServiceBus from version 7 to version 8.
+component: Core
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ - 8
+---
+
+## Obsoleted APIs
+
+### Gateway
+
+NServiceBus Gateway has been moved to a separate `NServiceBus.Gateway` package and all Gateway public API in NServiceBus obsoleted with the following warning:
+
+> Gateway persistence has been moved to the NServiceBus.Gateway dedicated package. Will be treated as an error from version 8.0.0. Will be removed in version 9.0.0.
+
+#### How to upgrade
+
+- Select the storage type to use in production. Supported storage types are SqlServer and RavenDB
```

Comment:
Perhaps avoid the work "storage type" since now users move away from using the storage type to "Install the wanted gateway persistence package and configure it to be used "?